nautobot/extras/tests/test_customfields.py

```diff
@@ -1740,68 +1771,14 @@ def test_filter_url(self):
 
     def test_filter_select(self):
         self.assertQuerysetEqual(
-            self.filterset({"cf_cf8": "Foo"}, self.queryset).qs,
-            self.queryset.filter(_custom_field_data__cf8="Foo"),
+            self.filterset({"cf_cf8": ["Foo", "AR"]}, self.queryset).qs,
+            self.queryset.filter(_custom_field_data__cf8__in=["Foo", "AR"]),
         )
         self.assertQuerysetEqual(
             self.filterset({"cf_cf8__n": ["Foo"]}, self.queryset).qs,
             self.queryset.exclude(_custom_field_data__cf8="Foo")
             | self.queryset.filter(_custom_field_data__cf8__isnull=True),
         )
-        self.assertQuerysetEqual(
```

Comment:
Did we lose existing filter capabilities? That seems less than desirable.

Reply:
We can't have it both ways, can we? If we allow `TYPE_SELECT` to be a multiple input filter from choices, then allowing `__isw` in the "Advanced" UI filter form makes no sense because the Value column still wants you to select from the provided choices.

However, to your concern, this still facilitates these filters being usable in the REST API or manually, just not intuitively in the dynamic filter form interface. I would say that for the v1 UI this is fine and we can attempt to fix this in the v2 UI while still allowing the filter to behave as-is.

Reply:
I'm assuming by the removal of these tests that this filtering (at the filterset level, not the UI level) is no longer working. Happy to be proven wrong.